tests/SingleImmutableValueObjectTest.php

```diff
@@ -4,26 +4,19 @@
 
 namespace NorseBlue\ValueObjects\Tests;
 
-use Exception;
 use NorseBlue\HandyProperties\Exceptions\PropertyNotMutableException;
 use NorseBlue\ValueObjects\Tests\Helpers\SingleImmutableVO;
 
 class SingleImmutableValueObjectTest extends TestCase
 {
     /** @test */
-    public function immutable_single_value_property_object_cannot_change_value()
+    public function immutable_single_value_property_object_cannot_change_value(): void
     {
         $subject = new SingleImmutableVO();
 
-        try {
-            $subject->value = 9;
-        } catch (Exception $e) {
-            $this->assertInstanceOf(PropertyNotMutableException::class, $e);
-            $this->assertEquals('value', $e->getProperty());
+        $this->expectException(PropertyNotMutableException::class);
+        $this->expectExceptionMessage("The 'value' property is immutable, cannot set the value '9'.");
```

Comment:
This way the assertions are not even equivalent. I'm making an assertion directly towards the `PropertyNotMutableException` class to see if it has the property value correctly set, which here it appears _disguised_.

What if the `handy-properties` package decides that it should change the exception message? The test would be unnecessarily broken.

I don't like to test again exception messages, it's to frail.